File: cli/src/utils/nano_id.rs
use rand::{Rng, rng};

pub fn get_nanoid(length: usize) -> String {
    const CHARSET: &[u8] = b"0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
    let mut r = rng();

    (0..length)
        .map(|_| {
            let idx = r.random_range(0..CHARSET.len());
            CHARSET[idx] as char
        })
        .collect()
}

File: packages/ocel/src/utils/nanoid.ts
import { customAlphabet } from "nanoid";

export function getNanoid(length = 7) {
  const alphabet =
    "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
  const nanoid = customAlphabet(alphabet, length);

  return nanoid();
}
